.eslintrc.json

```diff
@@ -22,6 +22,8 @@
       }
     ],
     "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/restrict-plus-operands": "off",
```

Comment:
Was complaining about `Date.now() + ms('1d')` even though both are numbers. 🤷‍♂️

Reply:
Oh? May have been my vscode. It was definitely weird, I was seeing both as `number` yet it wouldn't add!

Reply:
https://github.com/seamapi/react/issues/86